drydock/templates/drydock/k8s/ingress/cms.yml

```diff
@@ -33,6 +33,24 @@ spec:
   - host: {{ CMS_HOST }}
     http:
       paths:
+{%- for app_name in iter_mfes_per_service("cms") %}
+{%- if app_name == "authoring" %}
+      - pathType: Prefix
+        path: "/course-authoring"
+        backend:
+          service:
+            name: {% if DRYDOCK_BYPASS_CADDY -%}mfe{% else -%}caddy{% endif %}
+            port:
+              number: {% if DRYDOCK_BYPASS_CADDY -%}8002{% else -%}80{% endif %}
+{%- endif %}
```

Comment:
This should be an else statement

Reply:
We need to keep both paths to avoid broke bookmarks as [tutor does](https://github.com/overhangio/tutor-mfe/pull/225/files#diff-616f7c9c5749f902af0b1e89b1ac51154b62889f5a81b932c151fc4549cd837eR23-R27). We're adding this caddy change in mfe_extensions and here we redirect both `course-authoring` and `authoring` to mfe.